style-guide.md

```diff
@@ -0,0 +1,311 @@
+        ~~~
+    - This is a bullet.
+
+        ~~~ shell
+        $ command
+        ~~~
+
+2. This is a step.
+```
+
+### Notes, Warnings, & Tips
+
+When? note vs. note that vs. using call out
+
+Our docs use three classes of highlighted text:
+
+- [Tips](#tips)
+- [Notes](#notes)
+- [Warnings](#warnings)
+
+The text of notes, warnings, and tips must be formatted in HTML instead of Markdown/Kramdown.
+
+#### Tips
+
+Use tips to highlight nice-to-know pieces of information.
+
+For example, you might include a tip to our Github repo's Terraform scripts on the Google Cloud Engine deployment page. It's nice to know it's there, but doesn't clarify anything nor is it critical.
+
+To insert a tip, use the following code:
+
+~~~
+{{site.data.alerts.callout_success}} <tip text goes here> {{site.data.alerts.end}}
+~~~
+
+#### Notes
+
+Use notes to call attention to a piece of clarifying information; this information should not be crucial to accomplishing the task in the document.
+
+For example, you might use a note to let users know that the `DELETE` statement only deletes rows and that to delete columns you must use `ALTER TABLE`. This helps clarify `DELETE`'s purpose and point misguided users to the right place.
+
+To insert a note, use the following code:
+
+~~~
+{{site.data.alerts.callout_info}} <note text goes here> {{site.data.alerts.end}}
+~~~
+
+#### Warnings
+
+Use warning to express that a piece of information is critical to understand to prevent unexpected things from happening.
+
+For example, you might include a warning that using `CASCADE` in `DROP INDEX` drops dependent objects without warning. This is critical to prevent users from unexpectedly losing constraints or additional indexes.
+
+To insert a warning, use the following code:
+
+~~~
+{{site.data.alerts.callout_danger}} <warning text goes here> {{site.data.alerts.end}}
+~~~
+
+### Tables
+
+Use tables to display structured information in an easy-to-read format.
+
+To create a table, use pipes (`|`) between columns and at least 3 dashes (`-`) separating the header cells from the body cells. A return denotes the start of the next row. The text within each column do not need to align in order to be rendered correctly, and you can inline Markdown or HTML.
+
+when to move to html?
+
+We do not use outer pipes.
+
+Example:
+
+~~~
+   Term   |         Description         |     Example    
+----------|-----------------------------|----------------
+ `term_1` | This is a description.      | `3.14`         
+ `term_2` | This is also a description. | `"lola mcdog"`
+~~~
+
+### Text Format
+
+#### Bold
+
+When?
+
+Surround the text with two asterisks.
+
+Example:
+
+`**This is bold.**` or `Only **bold** will be bold.`
+
+#### Italics
+
+When?
+
+Surround the text with one underline.
+
+Example:
+
+`_This is italicized._` or `Only _italics_ will be italicized.`
+
+#### Monospace
+
+See [Inline Code](#inline-code).
+
+#### Quotations
+
+
+#### Underline
+
+When?
+
+### Version Tags
```

Comment:
Can you also provide guidance on where to put these tags? I spend a lot of time figuring out the correct placement in headings, sections, in a table, in examples, etc.